### PR TITLE
Add resetPasswordDto and return username from login method

### PR DESCRIPTION
**Pull request by MarrMika**
Return username to save in a session Storage

### CODE REVIEW

File: src/main/java/com/ita/if103java/ims/dto/ForgotPasswordDto.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import com.ita.if103java.ims.dto.transfer.NewData;
+
+import javax.validation.constraints.NotBlank;
+import java.io.Serializable;
+
+public class ForgotPasswordDto implements Serializable {
+    @NotBlank(groups = {NewData.class},
+        message = "Please, enter these data")
+    private String email;
```
Comment (ikrasWork) [inline src/main/java/com/ita/if103java/ims/dto/ForgotPasswordDto.java]: Why do you need to wrap single String field into object? For validation only?

Comment (MarrMika) [reply src/main/java/com/ita/if103java/ims/dto/ForgotPasswordDto.java]: Yes, and I think it is better to get data in  JSON from UI

File: src/main/java/com/ita/if103java/ims/controller/LoginController.java [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
```
Comment (ikrasWork) [inline src/main/java/com/ita/if103java/ims/controller/LoginController.java]: Avoid wildcards in import

Comment (MarrMika) [reply src/main/java/com/ita/if103java/ims/controller/LoginController.java]: Ok, thanks

**Pull request merged**